R/utils.R

```diff
@@ -82,6 +82,32 @@ name_dots = function(...) {
   vnames
 }
 
+col_as_int = function(tbl, cols) {
```

Comment:
I think we could re-use C function from there: https://github.com/Rdatatable/data.table/blob/master/src/nafill.c#L3
Ideally moving it to new `utils.c` file.
I can do it but best if we merge this foreign-remote based PR so we can push follow ups into Rdatatable remote.